pom.xml

```diff
@@ -72,7 +72,12 @@
         <hbase.version>1.3.1</hbase.version>
         <kafka.version>1.0.1</kafka.version>
         <nifi.version>1.8.0</nifi.version>
-        <hadoop.version>2.7.4</hadoop.version>
+        <hadoop.version>2.8.5</hadoop.version>
+        <aws.sdk.version>1.11.636</aws.sdk.version>
```

Comment:
Hmm... seems like including AWS is both a Good Thing and a thing which may cause us trouble...

Reply:
we need it to read s3a:// urls through hadoop - what trouble would it cause?